mender/install_mender.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+set -e
+
+if [ "$EUID" -ne 0 ]
+  then echo "Please run with sudo"
+  exit
+fi
+
+export SCRIPT_PATH=$(dirname $(readlink -f $0))
+
+function get_mender_connect()
+{
+   [ ! -d ${SCRIPT_PATH}/mender-connect ] && git clone -b 2.0.1 https://github.com/mendersoftware/mender-connect.git ${SCRIPT_PATH}/mender-connect
+   true
+}
+
+function install_mender_lib_from_package()
+{
+   echo "Installing lib dependencies for mender client and mender connect."
+   apt install ${SCRIPT_PATH}/libffi6_3.2.1-9_arm64.deb
+}
+
+function install_mender_client_from_package()
+{
+   echo "Installing mender client."
+   cp -R ${SCRIPT_PATH}/mender /etc/
+   cp -r ${SCRIPT_PATH}/lib/mender /var/lib/
```

Comment:
Following suggestion on 
https://github.com/perimeter-inc/system-dependencies/pull/4/files#r808221249

```suggestion
    echo -e "${DEVICE_TYPE}" >> ${SCRIPT_PATH}/lib/mender/device_type
   cp -r ${SCRIPT_PATH}/lib/mender /var/lib/
```